docs/code-standards.md

```diff
@@ -0,0 +1,35 @@
+# Code & Documentation Standards
+
+This document details our goals for code quality, coverage and documentation standards. At the time
+of this writing, these are standards that all of our code may not meet. We should always try to 
+improve our code to meet them.
+
+## Code Quality And Documentation Standards
+
+We aim to write clear code that is easy to read and comprehend. In particular, that goal means the
```

Comment:
Probably want to link to https://github.com/kubernetes/community/blob/master/contributors/devel/coding-conventions.md here